engine/lib/phx/src/engine/mod.rs

```diff
@@ -349,19 +349,27 @@ impl Engine {
                             .lua
                             .load(&*entry_point_path)
                             .exec()
-                            .expect("Cannot execute entry point script");
+                            .unwrap_or_else(|e| {
+                            panic!("Error executing the entry point script: {}", e);
```

Comment:
I had to do this on purpose, because `expect` ends up formatting the error like
```
Error executing the entry point script: ExecutionError{"something happened\nstack trace:\n  [main.lua] main_func\n.  etc\n"}
```

Reply:
So we need to explicitly format the error object using the Display trait instead of the Debug trait.